ebpfsvc/rpc_api.cpp

```diff
@@ -21,6 +21,9 @@ ebpf_server_verify_and_load_program(
         return EBPF_INVALID_ARGUMENT;
     }
 
+    // Set the handle of program being verified in TLS.
```

Comment:
```suggestion
    // Set the handle of program being verified in thread-local storage.
```
Since TLS is a protocol we used in the SGX enclave test, TLS as an acronym might be confusing.